packages/discovery-react-components/src/components/CIDocument/types.ts

```diff
@@ -46,7 +46,11 @@ export interface ItemMap {
   bySection: any;
 }
 
-export type EnrichedHtml = EnrichedHtmlContract | EnrichedHtmlInvoice | EnrichedHtmlPurchaseOrder;
+export type EnrichedHtml =
+  | EnrichedHtmlContract
+  | EnrichedHtmlInvoice
+  | EnrichedHtmlPurchaseOrder
+  | any;
```

Comment:
adding `any` kind of defeats the purpose of having the type at all, no?

Reply:
So the EnrichedHtml can have other enrichments as well other than `contract`, `invoices`, `purchaseOrder`. eg. `tables` etc. I am not really sure what would all the other values would be so had to put any

Reply:
If we don't know what the other values will be, then we won't be referencing those anywhere else. Therefore, having explicit types (not `any`) will be fine from a typescript point-of-view.

If we are referencing `tables` somewhere else, then that should be part of this type as well.